buildmaster/ATLAS_2JET_7TEV_R06/filter_bugged.py

```diff
@@ -0,0 +1,53 @@
+from validphys.api import API
+import numpy as np
+import yaml
+
+def decompose_covmat(covmat):
+     """Given a covmat it return an array sys with shape (ndat,ndat)
+     giving ndat correlated systematics for each of the ndat point.
+     The original covmat is obtained by doing sys@sys.T"""
+
+     lamb, mat = np.linalg.eig(covmat)
+     sys = np.multiply(np.sqrt(lamb), mat)
+     return sys
+
+
+def filter_ATLAS_2JET_7TEV_R06_uncertainties_bugged():
+    """
+    - read covmat from old CommonData format
+    - compute artificial uncertainties and save them to
+      .yaml file
+    """
+    inp = dict(
+     dataset_input={'dataset': 'ATLAS_2JET_7TEV_R06'},
+     theoryid=200,
+     use_cuts="internal"
+     )
+
+    covmat = API.covmat_from_systematics(**inp)
```

Comment:
Maybe instead of having artificial anonymous systematics for everything we can do more accurately than this by following what `validphys.covmats.construct_covmat` does, i.e reading off
```
d = Loader().check_commondata("ATLAS_2JET_7TEV_R06").load()
cd.systematic_errors()
cd.stat_errors
```

ideally we would want to keep things like the atlas lumi correlation.

?